.github/workflows/build.yml

```diff
@@ -30,19 +38,19 @@ jobs:
             symfony-version: ''
 
           # MacOS on latest PHP only
-          - php: 8.2
+          - php: 8.4
             os: macos-latest
             composer-mode: update
             symfony-version: ''
 
           # Windows on latest PHP only
-          - php: 8.2
+          - php: 8.4
             os: windows-latest
             composer-mode: update
             symfony-version: ''
 
           # Symfony LTS jobs:
-          - php: 8.2
+          - php: 8.4
```

Comment:
I think it is best if this job remains with PHP 8.2

Reply:
done